doc/sphinx/source/project/release-notes.rst

```diff
@@ -1,41 +1,61 @@
-.. release-notes:
+.. ngraph/release-notes:
 
 Release Notes
 #############
 
-This is |release|.
+The latest |version| download below.
 
+* `Format .zip`_ 
+* `Format tar.gz`_ 
 
+See also: https://github.com/NervanaSystems/ngraph/releases for previous versions. 
 
 
+CHANGELOG |release|
+-------------------
 
-CHANGELOG
-=========
+nGraph v0.17.0-rc.0
+~~~~~~~~~~~~~~~~~~~
 
-(Last updated September 2018)
++ Allow negative padding in more places
++ Add code generation for some quantized ops
++ Preliminary dynamic shape support
++ initial distributed ops
 
-This release focuses on accelerating deep learning inference workloads on 
-Intel® Xeon® (CPU processor) and has the following key features: 
 
-* Out-of-box installation experience for TensorFlow*, MXNet*, and ONNX.
-* Validated optimizations for 17 workloads each on both TensorFlow and MXNet, 
-  as well as 14 for ONNX.
-* Support for Ubuntu 16.04 (TensorFlow, MXNet and ONNX).
-* Support for OSX 10.13.x (buildable for TensorFlow and MXNet).
 
-This |version| release includes optimizations built for popular workloads 
-already widely deployed in production environments. These workloads cover 
-the following categories:
+nGraph v0.16.0-rc.3
+~~~~~~~~~~~~~~~~~~~~
 
```

Comment:
Since rc.3 has rc.2 and earlier in it, we can skip rc.2 and rc.1 here.

Reply:
Done